recipes/entt/3.x.x/conanfile.py

```diff
@@ -53,26 +51,28 @@ def configure(self):
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        extracted_dir = "entt-" + self.version
-        os.rename(extracted_dir, self._source_subfolder)
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
 
     def _configure_cmake(self):
-        cmake = CMake(self)
-        cmake.definitions["BUILD_TESTING"] = "OFF"
-        cmake.definitions["USE_LIBCPP"] = "OFF"
-        cmake.configure(
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["USE_LIBCPP"] = False
+        self._cmake.definitions["USE_ASAN"] = False
+        self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["BUILD_DOCS"] = False
+        self._cmake.configure(
             source_folder=self._source_subfolder,
             build_folder=self._build_subfolder
         )
-        return cmake
+        return self._cmake
 
     def build(self):
```

Comment:
If this is a header-only, why do we need the build step?

Reply:
It has targets. Kitten recipe does the same thing (and maybe the c++17 in Kitten recipe could be used in entt recipe).
Honestly, we could just copy src dir and don't even care about CMake.

Reply:
ok, we will take care of that in another PR. Thanks!